### PR TITLE
Add locator.hover

### LINKED ISSUE #342: locator.hover([options])
The [`locator.hover([options])`](https://playwright.dev/docs/api/class-locator#locator-hover) method hovers on the element that matches the locator's selector (with strict mode).

### PR DESCRIPTION
**Pull request by inancgumus**
Closes #342.

### CODE REVIEW

File: api/locator.go [hunk 1]
```diff
@@ -64,4 +64,7 @@ type Locator interface {
 	// Type text on the element found that matches the locator's
 	// selector with strict mode on.
 	Type(text string, opts goja.Value)
+	// Hover the element found that matches the locator's selector with
+	// strict mode on.
```
Comment (imiric) [inline api/locator.go:68]:
```suggestion
	// Hover moves the pointer over the element that matches the locator's
	// selector with strict mode on.
```

File: common/frame.go [hunk 2]
```diff
@@ -1038,28 +1038,30 @@ func (f *Frame) Goto(url string, opts goja.Value) api.Response {
 	return resp
 }
 
-// Hover hovers an element identified by provided selector.
+// Hover the first element matches the selector.
```
Comment (imiric) [inline common/frame.go:1041]:
```suggestion
// Hover moves the pointer over the first element that matches the selector.
```

File: common/locator.go [hunk 3]
```diff
@@ -526,3 +526,25 @@ func (l *Locator) typ(text string, opts *FrameTypeOptions) error {
 	opts.Strict = true
 	return l.frame.typ(l.selector, text, opts)
 }
+
+// Hover the element found that matches the locator's selector with
+// strict mode on.
```
Comment (imiric) [inline common/locator.go:531]:
```suggestion
// Hover moves the pointer over the element that matches the locator's selector
// with strict mode on.
```

File: tests/static/locators.html [hunk 4]
```diff
@@ -30,6 +30,9 @@
         document.querySelector('#inputCheckbox').addEventListener(
             'change', e => { window.check = e.currentTarget.checked; }, false
         );
+        document.querySelector('#inputText').addEventListener(
+            'mousemove', e => { result = true; }, false
```
Comment (imiric) [inline tests/static/locators.html:34]:
```suggestion
            'mousemove', e => { window.result = true; }, false
```

Same for the `click` handler.

**Review by imiric — CHANGES_REQUESTED** [review]

File: tests/static/locators.html (hunk 4 above)
Comment (inancgumus) [reply tests/static/locators.html]: Thanks, see https://github.com/grafana/xk6-browser/pull/400/commits/c3a90bd8f2ebad00c01ffcec29718e30711461d6.

File: common/locator.go (hunk 3 above)
Comment (inancgumus) [reply common/locator.go]: Thanks, see https://github.com/grafana/xk6-browser/pull/400/commits/c3a90bd8f2ebad00c01ffcec29718e30711461d6.

File: common/frame.go (hunk 2 above)
Comment (inancgumus) [reply common/frame.go]: Thanks, see https://github.com/grafana/xk6-browser/pull/400/commits/c3a90bd8f2ebad00c01ffcec29718e30711461d6.

File: api/locator.go (hunk 1 above)
Comment (inancgumus) [reply api/locator.go]: Thanks, see https://github.com/grafana/xk6-browser/pull/400/commits/c3a90bd8f2ebad00c01ffcec29718e30711461d6.

**Review by ankur22 — APPROVED** [review]

**Pull request merged**